UserShiftsApiService/UserShiftsApiService/Program.cs

```diff
@@ -15,12 +16,22 @@
 });
 
 builder.Services.AddSingleton<IGreetingService, GreetingService>();
+builder.Services.AddScoped<ISaveNewEmployeeService, SaveNewEmployeeService>();
```

Comment:
be more specific here that this service is for auth0 callbacks

Reply:
and more generic that it has nothing to do with employees or saving them. its basically the controller that should interact with auth0 only.